cross-chain/solana/scripts/deploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eo pipefail
+
+# Setting env variables in the current bash shell
+source solana.env
+
+[ -z "$CLUSTER" ] && {
+  echo "'--cluster' option not provided" >&2
+  help
+  exit 1
+}
+
+[ -z "$AUTHORITY" ] && {
+  echo "'AUTHORITY' env var is not set" >&2
+  exit 1
+}
+
+echo "Building workspace for cluster: $CLUSTER ..."
+anchor build --provider.cluster $CLUSTER
+
+echo "Syncing the program's id ..."
+anchor keys sync
+
+echo "Building workspace again to include new program ID in the binary ..."
+anchor build --provider.cluster $CLUSTER
+
+echo "Deploying program(s) for cluster: $CLUSTER ..."
+anchor deploy --provider.cluster $CLUSTER --provider.wallet $AUTHORITY
```

Comment:
I would avoid using `anchor deploy` and use `solana program deploy` instead. The anchor deploy assumes you have a keypair in the `target` directory, which is a bit clunky from my experience.

Reply:
Sounds good, using `solana program deploy` instead. Pls see [3882cc4](https://github.com/keep-network/tbtc-v2/pull/688/commits/3882cc4e49a9240d7420f4d825bc4005a18dd4df)